extensions/VisualEditor/wikia/modules/ve/ui/styles/images/icons/image.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I used the optimizer with the same "conservative" settings for both this and video2.svg.
I think this icon is just more complex than the video icon.